ci-makefile.toml

```diff
@@ -0,0 +1,25 @@
+[env]
```

Comment:
I moved this into a dedicated file since `cat`ing it together wouldn't work on a Windows shell. Is there a better alternative?